README.md

```diff
@@ -295,6 +296,17 @@ let client = Graph.Client(
 )
 ```
 
+If your store supports multiple languages, Storefront API can return translated resource types and fields. Learn more about [translating content](https://help.shopify.com/en/api/guides/multi-language/translating-content-api).
+
+```swift
+// Initializing a client to return translated content
+let client = Graph.Client(
+	shopDomain: "shoes.myshopify.com",
+	apiKey:     "dGhpcyBpcyBhIHByaXZhdGUgYXBpIGtleQ",
+        locale:     Locale.current
```

Comment:
@DrTeePot I will leave this because the end product seems fine. Below is what it looks like when I apply your suggestion. 
![image](https://user-images.githubusercontent.com/655059/75287288-197ef000-57e8-11ea-99f4-d3eb0234ade1.png)

Look at the preview or richtext view for feedbacks on formatting